components/ai_chat/browser/ai_chat_tab_helper.cc

```diff
@@ -139,6 +144,12 @@ void AIChatTabHelper::MaybeGeneratePageText() {
     return;
   }
 
+  const GURL url = web_contents()->GetURL();
+
+  if (url.SchemeIs(kBraveScheme) || url.SchemeIs(kChromeScheme)) {
```

Comment:
Can we just make this a whitelist of schemes instead? (http, https, file, data maybe?)

For instance we probably also don't want to summarize protocols like chrome-extension://.

Reply:
great point, changed